### PR TITLE
Add basic interactables through an interface

### PR DESCRIPTION
**Pull request by Westlanderz**
This PR adds the ability to make interactables by creating a new script for that interactable and adding it to a prefab.

### CODE REVIEW

**Review by drfdaniel — DISMISSED** [review]
LGTM, all is logical IMO, idk about classes and stuff though. But I trust your judgment

File: Assets/Scripts/Core/Interactables/Crate.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            // Calculate the distance between the two units. But due to the -1 we can attack diagonal units, but also sometimes 1 node extra on the range.
+            int nodesBetweenPlayers = GridCombatSystem.Instance.CalculatePoints(unit.GetPosition(), transform.position).Count - 1;
+			return nodesBetweenPlayers <= range_;
+        }
+
+        public int GetCost() {
+            return cost_;
+        }
+    }
+}
```
Comment (Dennisbonke) [inline Assets/Scripts/Core/Interactables/Crate.cs:26]: Missing newline at the end of file.

File: Assets/Scripts/Core/Interactables/IInteractable.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+namespace OperationBlackwell.Core {
+    public interface IInteractable {
+        void Interact();
+        bool IsInRange(CoreUnit unit);
+        int GetCost();
+    }
+}
```
Comment (Dennisbonke) [inline Assets/Scripts/Core/Interactables/IInteractable.cs:7]: Missing newline at the end of file.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
Code looks good, 2 minor code style issues.

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**